packages/react-components/react-nav-preview/src/components/NavSubItemContext.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { NavItemValue } from './NavContext.types';
+
+// This context is analogous to AccordionItemContext
+
+export type NavSubItemContextValue = {
+  open: boolean;
+  value: NavItemValue;
```

Comment:
nit: NavItemValue is unknown type, should we enforce a string to simplify use & enable use as non-iterative ID for future accessibility usecases?

Reply:
discussed offline - this is a pattern set in a couple other components. Needs a blog posty document though :-)